### ISSUE TITLE
Data for question classification as search-free, image-, text-, mixed-search-required

### ISSUE THREAD

**Issue by vishwassathish** [EvolvingLMMs-Lab/multimodal-search-r1#12]
Hi,

Excellent work with this paper! Thank you for releasing the code and dataset. 

I was going through the [released dataset ](https://huggingface.co/datasets/lmms-lab/FVQA/blob/main/README.md) and was not able to find the question classification against each row that was mentioned in Page 6 (Search Balancing) of the [paper](https://arxiv.org/pdf/2506.20670). Each question from the training set is classified to one of the 4 classes: search-free, image-search-required, text-search-required, mixed-search-required.

Wondering if this additional information would be updated in the released dataset.

Thank you once again.

**Comment by vvukimy** [comment]
Thank you for your interest in our work.

We have now added the 'search-free' and 'search-required' labels to the dataset. During the data selection process, we unfortunately did not preserve the per-question detailed search behavior labels (e.g., img-, txt-search-required), so all search-required variants have been merged into a single ;search-required' category.

Due to certain copyright considerations, the number of released samples is slightly different from what was reported in the paper. We appreciate your understanding.

**Comment by vishwassathish** [comment]
Thank you for the updated dataset!
